src/pages/Login.vue

```diff
@@ -162,12 +162,12 @@
         :buttonClass="resendOTPButtonClass"
         class="mt-2"
         :isDisabled="submitOTPIconConfig.enabled"
-        v-if="requestedOtp && !resentOtp"
+        v-if="requestedOtp && !counting"

```

Comment:
this will change accordingly as you'll need to always show this resend button but the styling and the text based on whether the counter is running or not.